src/components/movie.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import firebase from '../firebaseConfig'
+
+function Movie(props){
+
+    const [coverImage, setCoverImage] = useState('')
+    const [dateOfEntry, setDateOfEntry] = useState('')
+    const [description, setDescription] = useState('')
+    const [name, setName] = useState('')
+    const [rating, setRating] = useState('')
+
+    useEffect(() => {
```

Comment:
where is useEffect() being called?

Reply:
useEffect is the same as component did mount for functional components

Reply:
got it